src/mrack/outputs/ansible_inventory.py

```diff
@@ -38,30 +39,61 @@ def copy_meta_attrs(host, meta_host, attrs):
             host[f"meta_{attr}"] = val
 
 
-def find_group(inventory, group_names, top=True):
-    """Find first matching group in inventory which has one of the provided names."""
-    if not inventory:
-        return None
+def ensure_all_group(inventory):
+    """Ensure that inventory has group "all" with both "hosts" and "children"."""
+    if "all" not in inventory:
+        inventory["all"] = {
+            "children": {},
+            "hosts": {},
+        }
+    all_group = inventory["all"]
+    if "children" not in all_group:
+        all_group["children"] = {}
+    if "hosts" not in all_group:
+        all_group["hosts"] = {}
+    return all_group
+
 
+def get_group(inventory, groupname):
+    """Get group from inventory, return group or None."""
     groups = inventory.keys()
-    for group in groups:
-        if group in group_names:
-            return inventory[group]
+    found = None
+    for g_name in groups:
+        group = inventory[g_name]
+        if g_name == groupname:
+            found = group
+        else:
+            if "children" in group:
+                found = get_group(group["children"], groupname)
+        if found:
+            break
+    return found
 
-    for group in groups:
-        g = inventory[group]
 
-        if "children" not in g:
-            continue
+def add_to_group(inventory, groupname, hostname):
+    """
+    Find a group in inventory layout and add a host to it.
 
-        found = find_group(g["children"], group_names, top=False)
-        if found:
-            return found
+    Returns True if host was added to a group, False if not.
```

Comment:
Just a note that it does not return `False`/`True` Could you please update the doctext?